framework/src/meshmodifiers/SidesetAroundSubdomainUpdater.C

```diff
@@ -160,10 +173,22 @@ SidesetAroundSubdomainUpdater::finalize()
   {
     for (const auto & [elem_id, side] : sent_data)
     {
-      _boundary_info.remove_side(mesh.elem_ptr(elem_id), side, _boundary_id);
+      const auto elem = mesh.elem_ptr(elem_id);
+      if (!_mesh.getConstructNodeListFromSideList())
+        mooseDoOnce(mooseWarning(
+            "The user has selected 'construct_node_list_from_side_list' as false, but "
+            "SidesetAroundSubdomainUpdate is building node lists from the side lists."));
```

Comment:
move this next to where the offending call is

maybe it should be a boolean option instead of being forced into it